server/app/com/xsn/explorer/services/CurrencyService.scala

```diff
@@ -29,6 +29,13 @@ object Currency extends Enum[Currency] {
   final case object NZD extends Currency("NZD")
   final case object TRY extends Currency("TRY")
   final case object UAH extends Currency("UAH")
+  final case object UAH extends Currency("AUD")
```

Comment:
I'm surprised that the code compiles because `UAH` is repeated many times.

Reply:
Oh jeep, i had one job... Sorry man i guess i was tired and didnt really check... i will correct that clown mistake